Tasks/AndroidSigningV3/androidsigning.ts

```diff
@@ -0,0 +1,135 @@
+import * as path from 'path';
+import * as Q from 'q';
+import * as tl from 'vsts-task-lib/task';
+
+const findAndroidTools = (tool: string): string => {
+    const androidHome = tl.getVariable('ANDROID_HOME');
+    if (!androidHome) {
+        throw tl.loc('AndroidHomeNotSet');
```

Comment:
I think it's better to get in the habit of using this pattern: `throw new Error(tl.loc('AndroidHomeNotSet')`.  That way we get stacktracing preserved.

Reply:
Several places in this file could use that improvement.